cmd/jujud/agent/checkconnection.go

```diff
@@ -56,7 +57,7 @@ func (c *checkConnectionCommand) Info() *cmd.Info {
 // Init is part of cmd.Command.
 func (c *checkConnectionCommand) Init(args []string) error {
 	if len(args) == 0 {
-		return &agenterrors.FatalError{"agent-name argument is required"}
+		return fmt.Errorf("agent-name argument is required %w", errors.Hide(agenterrors.FatalError))
```

Comment:
If I'm reading this right, this will print an extra space at the end of the error message. So `"agent-name argument is required%w"` would be better.